multidict/__init__.pyi

```diff
@@ -63,9 +63,9 @@ class MutableMultiMapping(
     @abc.abstractmethod
     def popall(self, key: _S, default: _D) -> Union[List[_T], _D]: ...
 
-class MultiDict(MutableMultiMapping[_S, _T], Generic[_S, _T]):
+class MultiDict(MutableMultiMapping[str, _T]):
     def __init__(self, arg: _Arg[_S, _T] = ..., **kwargs: _T) -> None: ...
```

Comment:
I think this fails due to the `_S`. mypy now expects only 1 type in the annotation, but also wants 2 types to be defined (_S, _T), which appears to be impossible.